src/Lykke.Service.BlockchainWallets/Controllers/WalletsController.cs

```diff
@@ -3,16 +3,18 @@
 using System.Linq;
 using System.Net;
 using System.Threading.Tasks;
+using Castle.DynamicProxy.Generators;
 using Lykke.Common.Api.Contract.Responses;
 using Lykke.Service.BlockchainWallets.Core.Services;
 using Lykke.Service.BlockchainWallets.Models;
 using Microsoft.AspNetCore.Mvc;
 
 namespace Lykke.Service.BlockchainWallets.Controllers
 {
-    [Route("api/wallets/{integrationLayerId}/{integrationLayerAssetId}")]
+    [Route("api/wallets")]
     public class WalletsController : Controller
     {
+        public const string _routeSuffix = "{integrationLayerId}/{integrationLayerAssetId}";
```

Comment:
Why this constant is public? Also, suggested naming for this constant is `RouteSuffix`

Reply:
fixed also